lib/omniauth/strategies/drip.rb

```diff
@@ -17,7 +17,7 @@ class Drip < OmniAuth::Strategies::OAuth2
         :token_url => 'https://www.getdrip.com/oauth/token'
       }
 
-      uid { access_token.client.id }
+      uid { raw_info['accounts'][0]['id'] }
```

Comment:
Gotcha... but isn't it the account that we're authorizing? For example, I login to Drip using one email address, but I have two Drip accounts. If the user email is used as the identifier, how would we know which Drip account it's linking to? 

This is what it looks like when I go to the authorize url: 
<img width="345" alt="contentupgrade_me" src="https://cloud.githubusercontent.com/assets/234/24476087/b52d38fa-1497-11e7-93d1-8d98b91dfc9a.png">

If I authorize using my email address, and select the first Drip account, my user_info looks like this: 

    (byebug) user_info["users"]
    [{"email"=>"ryan@mysmallidea.com", "name"=>"Ryan Heneise", "time_zone"=>"America/Chicago"}]

... and the account info looks like this: 

    (byebug) raw_info['accounts']
    [{"id"=>"xxxx222", "href"=>"https://api.getdrip.com/v2/accounts/xxxx222", "name"=>"mysmallidea.com", "url"=>"mysmallidea.com" ...}]


Now if I want to connect the second Drip account (under the same user login), my user info looks the same: 

    (byebug) user_info["users"]
    [{"email"=>"ryan@mysmallidea.com", "name"=>"Ryan Heneise", "time_zone"=>"America/Chicago"}]

... but the account info has the unique ID that I can use to identify the account: 
    (byebug) raw_info['accounts']
    [{"id"=>"xxx337", "href"=>"https://api.getdrip.com/v2/accounts/xxxx337", "name"=>"memberman.com", "url"=>"www.memberman.com", "default_from_name"=>"Ryan Crispin Heneise", "default_from_email"=>"hello@memberman.com" ...}]

In this case wouldn't it make more sense to use the account ID rather than the user email as the UID?